infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/LogReplicationConfig.java

```diff
@@ -59,12 +60,21 @@ public class LogReplicationConfig {
             PROTOBUF_TABLE_ID
     ));
 
+    // Suite of utility methods for updating the configuration
+    private LogReplicationConfigManager configManager;
+
     // Unique identifiers for all streams to be replicated across sites
     private Set<String> streamsToReplicate;
 
+    // Mapping from stream ids to their fully qualified names.
+    private Map<UUID, String> streamsIdToNameMap;
```

Comment:
I think as a follow-up PR we should consider moving this functionality to a place within the TableRegistry itself - that way we will avoid duplicating memory and also help other use cases as well.
For now can we please leave a comment saying this should be moved into TableRegistry?